### ISSUE TITLE
Alt text on images not visible

### ISSUE THREAD

**Issue by FediVideos** [mastodon/mastodon-android#100]
There doesn't seem to be any way to view the alt text image descriptions?

This can contain important content, it would be good to have (for example) a long press expose the alt text, at least as an option in the context menu.

**Comment by grishka** [comment]
There is currently no way to _have it displayed on screen_, but you can hear it if you use a screen reader.

**Comment by realpixelcode** [comment]
How about displaying a native Android toast popup with the alt text after long-pressing the image?

**Comment by sk22** [comment]
^ mentioned this issue by accident, sorry about this!

**Comment by andria-dev** [comment]
I think a feature like the one described in the Mastodon web client would be great to add to the Android client.
https://github.com/mastodon/mastodon/issues/18394

**Comment by RiderExMachina** [comment]
> How about displaying a native Android toast popup with the alt text after long-pressing the image?

You mean like how Tusky does it?

**Comment by realpixelcode** [comment]
> > How about displaying a native Android toast popup with the alt text after long-pressing the image?
> 
> You mean like how Tusky does it?

Yes, exactly. Tusky also displays the alt text in beneath the image in a “text container”; that would be an option too.

**Comment by sk22** [comment]
> Yes, exactly. Tusky also displays the alt text in beneath the image in a “text container”; that would be an option too.

well, that's what my PR already implements :D https://github.com/mastodon/mastodon-android/pull/129

![Screenshot_2022-11-19-23-04-04-535_org joinmastodon android sk](https://user-images.githubusercontent.com/6217438/202873208-95c22d6e-8014-4a4b-8e03-fd5cfbedc768.jpg)

**Comment by realpixelcode** [comment]
Great! Love it! :)
